src/chapters/sql-part2/exercises.rst

```diff
@@ -0,0 +1,97 @@
+Exercises: SQL, Part 2
+=======================
+
+In the previous chapter, we set up a database to store information about seeds. Let's expand that idea now that we know all about foreign keys!
+
+Create a New Schema
+-------------------
+
+To get started, let's set up a new schema in MySQL Workbench and call it ``garden``.
+In the last chapter, we learned we can do this by clicking on *New Schema*, naming it ``garden``, accepting the defaults, and clicking *Apply*.
```

Comment:
Now that lesson 14 is merged, we can link to the chapter section that describes in detail how to create a new schema.

Reply:
Done! Thanks!